manifests/charts/gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -53,10 +53,7 @@ spec:
     spec:
 {{- if not $gateway.runAsRoot }}
       securityContext:
-        runAsUser: 1337
```

Comment:
This breaks ephemeral containers: https://istio.io/latest/docs/ops/configuration/security/harden-docker-images/#debugging.

Before: container comes up as user 1337, works fine.
After: container comes up as user of the dockerfile, which could be `0` (it is for istio/base - others can be used), which is blocked by `runAsNonRoot: true`

Reply:
Fixed